api/envoy/admin/v3alpha/BUILD

```diff
@@ -17,3 +29,29 @@ api_proto_package(
         "@com_github_cncf_udpa//udpa/annotations:pkg",
     ],
 )
+
+api_proto_package(
```

Comment:
Shall we generate this to a different tree? I think that will keep filename identical and really envoy internal (i.e. without exposing it to data-plane-api)? WDYT?

Reply:
Yeah, that's an interesting idea, but it's going to be tricky to do this without also duplicating all the v2 protos, since otherwise we need to overlay (conflicting v3 files, the shadow and original) over the base `@envoy_api`. Do you see a cleaner way than that?

Reply:
If we are OK with changing filenames though as in this PR, I think we can move to a separate tree and overlay this. We shold have `@envoy_api_shadow` local repository and `@envoy_api`, both with an `envoy/` tree.

Reply:
Just duplicate everything including v2 protos in `@envoy_api_shadow`? (or vice versa, export one without shadow to data-plane-api).

Reply:
Done, PTAL, thanks.